src/components/form/form.svelte

```diff
@@ -0,0 +1,74 @@
+<script lang="ts">
+  import BasicsSection from '@src/components/form/basicsSection.svelte';
+  import WorkSection from '@src/components/form/workSection.svelte';
+  import EducationSection from '@src/components/form/educationSection.svelte';
+  import SkillsSection from '@src/components/form/skillsSection.svelte';
+
+  function printResume() {
+    window.print()
+  }
+</script>
+
+<div class="form">
+  <section id="import">
+    <h1>Import Resume</h1>
+    <p class="full-width">MegaResume uses the JSON Resume format to import your resume. <a
+        id="what-is-json-resume" href="https://jsonresume.org/schema/">Learn more about JSON Resume.</a></p>
+    <input id="import-json-input" type="file" accept=".json" style="display:none">
+    <button id="import-json-resume">Import JSON Resume</button>
+  </section>
+
+  <BasicsSection/>
+
+  <WorkSection/>
+
+  <EducationSection/>
+
+  <SkillsSection/>
+
+  <section id="themes">
+    <h1>Themes</h1>
+    <label class="has-checkbox-input">
+      <input type="radio" name="resume-theme" value="default" checked />
+      <span>Default</span>
+    </label>
+    <label class="has-checkbox-input">
+      <input type="radio" name="resume-theme" value="monospace" />
+      <span>Monospace</span>
```

Comment:
Replace the monospace style with a serif font style, which is just a copy of the previous style, but with the font-family replaced to a serif font.

Reply:
done